bundles/af-core/src/main/java/com/adobe/cq/forms/core/components/internal/models/v1/form/RecaptchaImpl.java

```diff
@@ -102,18 +107,28 @@ public Map<String, Object> getCaptchaProperties() throws GuideException {
 
         Map<String, Object> customCaptchaProperties = new LinkedHashMap<>();
         String siteKey = null;
+        String version = null;
+        String keyType = null;
         resource = resourceResolver.getResource(this.getPath());
         if (resource != null && cloudConfigurationProvider != null) {
             reCaptchaConfiguration = cloudConfigurationProvider.getRecaptchaCloudConfiguration(resource);
             if (reCaptchaConfiguration != null) {
                 siteKey = reCaptchaConfiguration.siteKey();
+                version = reCaptchaConfiguration.version();
+                keyType = reCaptchaConfiguration.keyType();
             }
         }
         customCaptchaProperties.put(RECAPTCHA_SITE_KEY, siteKey);
-        customCaptchaProperties.put(RECAPTCHA_URI, RECAPTCHA_DEFAULT_URL);
+        if (StringUtils.isNotEmpty(version) && version.equals("enterprise")) {
```

Comment:
Please make sure all code is covered, please resolve all codecov git comments

Reply:
Updated the codecov